generators/kubernetes/templates/istio/gateway.yml.ejs

```diff
@@ -20,7 +20,6 @@ apiVersion: <%= KUBERNETES_ISTIO_NETWORKING_API_VERSION %>
 kind: Gateway
 metadata:
   name: <%= app.baseName.toLowerCase() %>-gateway
-  namespace: <%= kubernetesNamespace %>
```

Comment:
There is still `<%= app.baseName.toLowerCase() %>.<%= kubernetesNamespace %>.<%= ingressDomain %>` could you please check for all occurance of `kubernetesNamespace` in templates

Reply:
I will do as soon as we decide with @PierreBesson if we follow this route